cmd/server/main.go

```diff
@@ -51,6 +51,9 @@ const IOSCert = "cert"
 const IOSPassPhrase = "passphrase"
 const IOSIsProduction = "isProduction"
 
+// time in seconds
+const UPSPollingInterval = 10
```

Comment:
IMHO 60 seconds is enough.

Reply:
The loop isn't that costly (http request on the same pod, number of variants usually small). With this in mind i would rather leave it at 10 seconds for better response times.